### ISSUE TITLE
[ADE Intake]: Debt Resolution: One VA Debt Letter - PDF Accessibility (Tag Markup)

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#104259]
> [!NOTE]
> **QUICK SUMMARY:** 
> - **Team going through the CC and looking to get a head start on accessibility.** 
>    - Mark up the example PDF to be provided by Denise with the expected tags so it is accessible and 508 compliant.
>    - Team is pushing for a PDF and they want it fast as they have an SES Goal for their leadership that says it will be **live** **in** **April**.  
> - _Child work ticket created to track the related PDF work: #104267._ 

## About Your Team

- **Product name**: One VA Debt Letter
- **Team name**: Debt Resolution - Financial Management
- **OCTO product owner**: @denisecoveyduc 
- **Product manager**: @Tom-Davis 
- **Designer(s)**:
- **Slack channel**: #financial-management
- **Dedicated accessibility specialist on your team** (if you have one):
- **Accessibility Champ** (if you have one): 

## About Your Work
> [!TIP]
> _Tell us briefly about your work and the outcomes you’re aiming to achieve._

#### Initiative
_Stakeholders at the Debt Management Center (DMC) have asked that we establish what we are calling the "One VA Debt Letter" to comply with legislation._

#### 
Desired outcome
_Mark up the example PDF to be provided by Denise with the expected tags so it is accessible and 508 compliant_

**From Denise Coveyduc:** 

> Our stakeholders at the Debt Management Center (DMC) have asked that we establish what we are calling the "One VA Debt Letter" to comply with legislation.
> 
> They are pushing for a PDF and they want it fast as they have an SES Goal for their leadership that says it will be live in April. 
> 
> I am fully expecting there to be a few iterations of this but I am hopeful we can achieve this goal for them.
> The team currently has a PDF implementation that I don't think will be accessible; at least not out of the gates.  They are exploring other avenues that are 508 accessible but given the timeline they are much more time intensive and complicated. I guess I am hoping we can launch with this initial version and iterate to improve a11y.
> 

#### Why would I ask for this?

> 1. This is a HUGE initiative for our partners who report to Congress on this every 3 months and they are getting a lot of flack that it hasn't been done
> 2. Veterans already have ALL of this information available on the debt portal today this is simply adding a button that will allow them to generate a PDF to download/save this consolidated view of the information.
> 3. Veterans will continue to receive the debt letters that notify them of overpayment debts and statements that notify them about medical copays. This online version is a combined view of that information and in addition.
> 4. It isn't something Veterans are asking for and I am not even sure how useful it will be - we may end up taking it down entirely depending on the response and congressional feedback.

  
## Collaborate with ADE
[**Review our menu of services**,](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/ADE/readme.md#services-we-offer) and check all the types of collaboration you're requesting, below:

### How would you like to collaborate with ADE?
- [ ] Research support [(if yes, please open an additional research ticket)](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/ade-accessibility-research.yaml)
    - [ ] Discuss research plan / conversation guide / recruitment of AT users
    - [ ] Advise on accessible prototypes
    - [ ] Schedule a [pilot session](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/ADE/assistive-tech-pilot-guide.md) with an ADE Accessibility Specialist
    - [ ] Schedule technical support during AT research sessions
    - [ ] Document accessibility-related issues during AT research sessions
    - [ ] Conduct research for your team
- [ ] VA 508 office audit review
- [ ] Accessibility test planning
- [ ] Support design discovery
- [ ] Review wireframes or mockups
- [x] PDF audit
- [ ] Staging / CodePen / Codespaces / Production audit
- [ ] Health check audit
- [ ] Training and education
- [ ] Spot check, general guidance
- [ ] **Something else:**  `insert type of collaboration`
- [ ] **Unsure?** _Check this box if you're not sure what kind of support you need, need multiple types of support, and/or want to meet with us. We'll schedule an introductory meeting to learn more about your team and project._

## Provide Your Supporting Artifacts 

> [!TIP]
> **Include artifacts like your product outline, user flows, mockups and prototypes, etc.** 
> - **Figma/Mural:** Please tell us which screens/board/section to look at so we don't leave comments on old stuff. 
> - **Staging Audits:** We will need staging user(s) and scenarios provided.

### Links to Artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. 

## Supporting Artifacts 
-  [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Debt%20Resolution/one_va_debt_letter/product%20outline.md)
- **Denise will add the example PDF that the ADE team member can work with** (received 3/3/2025): 
    -  **Current/ Draft PDF version** (that the Financial Management team is working on): [debt_letter_test_123.pdf](https://github.com/user-attachments/files/19060737/debt_letter_test_123.pdf) 
    -   **Reference**: This is the example they were given to replicate [example.pdf](https://github.com/user-attachments/files/19060755/example.pdf)
    - Please connect with [@digitaldrk](https://github.com/digitaldrk) and [@Dr-Pongo](https://github.com/Dr-Pongo) to discuss in more detail.
- [CC Kickoff Ticket #104021](https://github.com/department-of-veterans-affairs/va.gov-team/issues/104021)
- [ADE PDF GitHub Documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/11f3c67169a86b5c8602f109c2a1efdd3778737d/teams/ADE/create-accessible-PDF.md)

## Your Timeframe

> [!IMPORTANT]
> - We work with the majority product teams on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.
> - If you ever have any questions about the status of your ticket(s) please reach out to us in the [accessibility-help](https://dsva.slack.com/archives/C8E985R32) Slack channel by tagging Naomi Eke or Lakwi Perera.
> - We will notify VFS Teams via the weekly Team of Teams slide, if our bandwidth is impacted by holidays, PTO, planned research, larger projects, etc.
> - Additionally, feel free to access our [capacity calendar](https://calendar.google.com/calendar/embed?height=600&wkst=2&ctz=America%2FNew_York&mode=WEEK&src=Y19mOTgyMzUwZmM3M2E4MTYyYTEwODhhYjE0ZDAwZGQzNzMzNWIyYjU1OWE2MGQ2YjY3ZWI4MDIxNDE2ZjgxYzJhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20&color=%23616161).

### Is this work tied to a specific deadline?
- [x] Yes (**April 2025**)
- [ ] No
- [ ] I’m not sure

### Ticket Number
- [Kickoff Ticket #104021](https://github.com/department-of-veterans-affairs/va.gov-team/issues/104021)

### Which phases of the collaboration cycle have you completed?
_Select all that apply._
- [x] **CC Ticket Created** - _Involving ADE prior to starting!_ 
- [ ] Design intent
- [ ] Midpoint review
- [ ] Staging
- [ ] None. This initiative isn’t going through the collaboration cycle.

## Next Steps for Your Team
- [x] Save and submit this intake ticket.
- [x] c/o Martha in ADE internal channel. ~Start a new thread, and post a link to this ticket in the [accessibility-help](https://dsva.slack.com/archives/C8E985R32) Slack channel~ by tagging ~Naomi Eke~ or Lakwi Perera.
- [ ] If you're requesting accessibility support for user research, submit a research support request next: [Open an accessibility research support request](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/ade-accessibility-research.yaml).
- [ ] If you're creating an experimental design, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) 
- [ ] [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] Please don't close this intake when work is complete - ADE will close and track.

### Ticket Updates
- [x] Receive new intake ticket @sara-amanda and @lakper 
- [x] Respond to Slack message sent by ~VFS team~ Martha upon submission
- [ ] Reach out to the VFS team to set up Kickoff (if requested)
- [x] Reviewed by OCTO/Martha
- [x] Connect related tickets in this ticket
- [x] Create child tickets, as they apply to the work requested.
- [x] Create Quick Summary for top of ticket as a note.
- [x] Add additional supporting artifacts x 2 (3/3/2025)
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support)_ @sara-amanda checking and will add the dropdown. 
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Naomi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`

### Related Tickets (ADE Internal)
> [!IMPORTANT]
> #### Tasks are Now Sub-Issues
> - Related tickets are no longer being tracked in task lists, due to the planned depreceiation of tasks by GitHub in April 2025. 
> - Instead, related tickets will be connected to tickets via sub-issues.
> ##### Connecting Tickets
> - **To connect an existing ticket**, choose the drop-down from "create sub-issue" and select "add existing issue"
> - **To connect a new issue,** choose the drop-down from "create sub-issue" and select "create sub-issue," and select the respective template.

**Comment by sara-amanda** [comment]
Child ticket created @lakper [ADE Support] Debt Resolution: One VA Debt Letter - PDF Accessibility (Tag Markup) #104267

**Comment by denisecoveyduc** [comment]
Here is the current/draft PDF version that the Financial Management team is working on.  Please connect with @digitaldrk and @Dr-Pongo to discuss in more detail. 

[debt_letter_test_123.pdf](https://github.com/user-attachments/files/19060737/debt_letter_test_123.pdf)

**Comment by digitaldrk** [comment]
FYI, this is the example we were given to replicate

[example.pdf](https://github.com/user-attachments/files/19060755/example.pdf)

**Comment by sara-amanda** [comment]
Thank you @denisecoveyduc and @digitaldrk ! I have added these to our supporting artifacts. 😄 
cc: @lakper @artsymartha68

**Comment by lakper** [comment]
Closing as issue was tracked in child ticket and closed.
